### ISSUE TITLE
[DISCUSSSION] Do you think it would be worth having example implementations?

### ISSUE THREAD

**Issue by quantuminformation** [membership/membership.db#10]
I was thinking of creating some express (node) services to interact with this membership DB database. Do you think this would be a worthwhile exercise?
##

**Comment by koistya** [comment]
Yes, sure! If you like, we can join our efforts and integrate that db schema into [GraphQL Starter Kit](https://github.com/kriasoft/graphql-starter-kit) project that's based on Node.js and JavaScript. Feel free to get in touch on Skype: koistya

**Comment by quantuminformation** [comment]
Will be in touch soon!

**Comment by lifeiscontent** [comment]
@koistya @QuantumInformation any updates on this?
